control_panel_api/serializers.py

```diff
@@ -266,3 +269,41 @@ class Meta:
             'nickname',
             'name',
         )
+
+
+class ESBucketHitsSerializer(serializers.BaseSerializer):
+
+    def to_representation(self, aggregations):
+        accessed_by_sum_map = {}
+        accessed_by_type_map = {}
+
+        for result in aggregations.bucket_hits:
+            role_type, accessed_by = self._extract_fields(result.key)
+
+            accessed_by_type_map[accessed_by] = role_type
+
+            if accessed_by not in accessed_by_sum_map:
+                accessed_by_sum_map[accessed_by] = result.doc_count
+            else:
+                accessed_by_sum_map[accessed_by] += result.doc_count
+
+        results = [
+            {'accessed_by': k, 'count': v, 'type': accessed_by_type_map[k]}
+            for k, v in accessed_by_sum_map.items()
+        ]
+
+        sorted_by_sum = sorted(results, key=itemgetter('count'), reverse=True)
+
+        return sorted_by_sum
+
+    def _extract_fields(self, key):
+        match = re.search('alpha_(app|user)_([\w-]+)/', key)
```

Comment:
Luckily [GH](https://github.com/isaacs/github/issues/906) doesn't allow `_` in username or this RE would break for users with an underscore 🤓

Reply:
Shouldn't `alpha_` come from the `settings.ENV` otherwise it will only match keys in `alpha`.

Reply:
Well spotted, I meant to do that but forgot!